index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header id="header">
+        <div class="logo">
+            <a href="#"><h2>Bilal</h2></a>
+        </div>
+        <div class="hamBurger">
+            <span class="material-symbols-outlined">
+                menu
+                </span>
+        </div>
+    </header>
```

Comment:
- [ ] Dear @BilalLiaquat7, Notice your header color is a bit different from the original design. On the original design both the `logo` and the `hamburger`  icon are colored `#6070FF`. Please make the appropriate changes to your header section. 

### Your design 
![image](https://github.com/BilalLiaquat7/Portfolio/assets/69077061/e9bb61f8-db0e-44b6-9658-851337485977)

### Original design 
![image](https://github.com/BilalLiaquat7/Portfolio/assets/69077061/5889838e-62ef-4f6e-86b4-830df1152de4)